src/alire/alire-spawn.adb

```diff
@@ -40,13 +40,15 @@ package body Alire.Spawn is
 
       Command ("gprbuild",
                Empty_Vector &
+
                  "-gnatwU" &
```

Comment:
@mosteo I never noticed this one before. Maybe it can be removed?

Reply:
I don't really remember, but I see below that you found some need for it?

Reply:
I just moved the command that drifted away from the switch. But I don't why it is needed, or if it is needed,